azext_iot/operations/hub.py

```diff
@@ -1618,6 +1618,42 @@ def iot_hub_configuration_delete(
         handle_service_exception(e)
 
 
+def iot_hub_configuration_test_queries(
+    cmd,
+    target_condition=None,
+    custom_metric_queries=None,
+    hub_name=None,
+    resource_group_name=None,
+    login=None,
+    auth_type_dataplane=None,
+):
+    discovery = IotHubDiscovery(cmd)
+    target = discovery.get_target(
+        resource_name=hub_name,
+        resource_group_name=resource_group_name,
+        login=login,
+        auth_type=auth_type_dataplane,
+    )
+    resolver = SdkResolver(target=target)
+    service_sdk = resolver.get_sdk(SdkType.service_sdk)
+
+    try:
+        if custom_metric_queries:
+            custom_metric_queries = process_json_arg(custom_metric_queries, argument_name="content")
+
+        result = service_sdk.configuration.test_queries(target_condition, custom_metric_queries)
+
+        if not result.target_condition_error and not result.custom_metric_query_errors:
+            print("Validation passed!")
```

Comment:
What does string/message does the service return if the queries pass? What does the portal use to say that the queries worked?

Reply:
Also you can return the message if it makes sense

Reply:
this service call is a bit different, the return type is ConfigurationQueriesTestResponse. when it fails, it will not throw exception, the status code will be 200, so I need to check the value of target_condition_error  and custom_metric_query_errors. On the other hand, when validation passed, there is no message from service call, the target_condition_error  and custom_metric_query_errors will just be NONE. So i think it would be make sense for me to write a message telling user that the validation pass, otherwise nothing will be happened?😀

Reply:
do a return since it will be more useful for developers than a print (cause then they can do stuff like save the output vs having to save whatever prints to the console)

Reply:
got it! now it's returned😁